dagster/README.md

```diff
@@ -0,0 +1,34 @@
+# Dagster+
+
+## Overview
+
+Dagster is a next-generation open-source orchestration platform designed for building, running, and monitoring data asset workflows. The Dagster+ integration streams [event logs][1] to Datadog and includes an out-of-the-box log pipeline and dashboard.
+
+## Setup
+
+1. Click **Connect Accounts** to launch the OAuth flow and link your Dagster and Datadog accounts.
```

Comment:
Didn't catch this earlier, but where do you click from? Datadog integration tile or Dagster? If you have it, add a link here.